src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -7,23 +7,22 @@ import {
   Box,
   Text,
 } from "@chakra-ui/react";
-import { Link as RouterLink } from "react-router-dom";
 import { HomeIcon } from "@primer/octicons-react";
-import { Organisation } from "../../data/model/organisation";
```

Comment:
Can you please review these file paths. It looks like it's capitalised the files.

Reply:
Updated.